std/json.d

```diff
@@ -1706,12 +1706,14 @@ pure nothrow @safe unittest // issue 15884
 
 @safe unittest // issue 15885
 {
+    enum bool realInDoublePrecision = real.mant_dig == double.mant_dig;
+
     static bool test(const double num0)
     {
         import std.math : feqrel;
         const json0 = JSONValue(num0);
         const num1 = to!double(toJSON(json0));
-        version(Win32)
+        static if (realInDoublePrecision)
```

Comment:
Since when does DMD cover 64-bit reals? Most new Phobos releases come with a few new tests that fail exactly because they're not tested with 64-bit reals, so I keep on upstreaming relaxed unittests for LDC which actually supports and tests targets with double-precision reals.

If you consider the potential being-off-by-1-ulp as bug (for std.format and/or std.json?), the Win32 exception shouldn't have been here already in the first place. I'm just extending that exception to its true scope here.
Fixing std.format or std.json alone for lossless roundtrip is